Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -564,7 +564,11 @@ public bool CreateCoinStake(List<StakeTx> stakeTxes, Block block, ChainedBlock c
 
                 for (uint n = 0; (n < searchInterval) && !fKernelFound; n++)
                 {
-                    this.nodeLifetime.ApplicationStopping.ThrowIfCancellationRequested();
+                    if (this.nodeLifetime.ApplicationStopping.IsCancellationRequested)
+                    {
+                        this.logger.LogTrace("(-)[SHUTDOWN]:false");
```

Comment:
how should this log be interpreted?

Reply:
shutdown detected, leaving whatever i'm working on

Reply:
what's the false about? Returning false?

Reply:
that is return value of the method